src/versioninfo.cpp

```diff
@@ -289,7 +289,7 @@ void VersionInfo::parse(const QString& versionString, VersionScheme scheme,
     temp.remove(0, 1);
   }
 
-  QRegularExpression exp("^(\\d+)(\\.(\\d+))?(\\.(\\d+))?(\\.(\\d+))?");
+  static const QRegularExpression exp("^(\\d+)(\\.(\\d+))?(\\.(\\d+))?(\\.(\\d+))?");
```

Comment:
This should probably be extracted outside statically or in an anonymous namespace.